js/script-loader.js

```diff
@@ -0,0 +1,34 @@
+'use strict';
+
+// eslint-disable-next-line no-var
+var loadScript = (function () {
```

Comment:
We already have a script loader in dom.js
It'd be better if you improve it and then reuse it.

Reply:
`loadScript` is used in background page. Should we move `onDOMscripted` into script-loader.js?

Reply:
Yeah I guess that's the correct solution.